modules/client/src/tokenVoting/internal/constants.ts

```diff
@@ -107,3 +114,26 @@ export const INSTALLATION_ABI: MetadataAbiInput[] = [
       "The token mint settings struct containing the `receivers` and `amounts`.",
   },
 ];
+
+export const ERC20_FUNCTIONS = [
+  "function totalSupply() public view returns (uint256)",
+  "function balanceOf(address _owner) public view returns (uint256 balance)",
+  "function transfer(address _to, uint256 _value) public returns (bool success)",
+  "function transferFrom(address _from, address _to, uint256 _value) public returns (bool success)",
+  "function approve(address _spender, uint256 _value) public returns (bool success)",
+  "function allowance(address _owner, address _spender) public view returns (uint256 remaining)",
+];
```

Comment:
where did you get this info to be hardcoded?
please provide a comments explainging that, and feel free to add url ...etc